api/src/pcapi/core/offers/api.py

```diff
@@ -820,7 +824,9 @@ def edit_stock(
         validation.check_update_only_allowed_stock_fields_for_allocine_offer(updated_fields)
         stock.fieldsUpdated = list(set(stock.fieldsUpdated) | updated_fields)
 
+    changes = {}
     for model_attr, value in modifications.items():
+        changes[model_attr] = ColumnUpdateLog(old_value=getattr(stock, model_attr), new_value=value)
```

Comment:
Comme ça tu peux éviter d'avoir à passer par le dictionnaire `changes_data`

```suggestion
        changes[model_attr] = ColumnUpdateLog(old_value=getattr(stock, model_attr), new_value=value).dict()
```